py4DSTEM/process/utils/utils.py

```diff
@@ -152,17 +152,65 @@ def get_shift(ar1, ar2, corrPower=1):
     return xshift, yshift
 
 
-def get_shifted_ar(ar, xshift, yshift):
-    """
-	Shifts array ar by the shift vector (xshift,yshift), using the Fourier shift theorem (i.e.
-	with sinc interpolation).
+def get_shifted_ar(
+    ar, 
+    xshift, 
+    yshift,
+    periodic=True,
+    bilinear=False,
+    ):
+    """
+	Shifts array ar by the shift vector (xshift,yshift), using the either 
+    the Fourier shift theorem (i.e.	with sinc interpolation), or bilinear
+    resampling. Boundary conditions can be periodic or not.
+
+    Args:
+            ar (float):            input array 
+            xshift (float):        shift along axis 0 (x) in pixels
+            yshift (float):        shift along axis 1 (y) in pixels
+            periodic (bool):       flag for periodic boundary conditions
+            bilinear (bool):       flag for bilinear image shifts
+
+        Returns:
+            datacube                     datacube              
+            mask (bool):                 (optional) the bad pixel mask
```

Comment:
this doesn't appear to be an actual output, we should remove it

Reply:
It's possible I forgot to push that change, but I am pretty sure it's in there now . . .

Reply:
Nope, it isn't!

Reply:
fixed